### PR TITLE
Issue #1757. Get and set issue model stateClass from milestones

### PR DESCRIPTION
**Pull request by miketaylr**
(and a few other small fixes)

r? @MDTsai

### CODE REVIEW

File: webcompat/static/js/lib/issues.js [hunk 1]
```diff
@@ -92,14 +92,6 @@ md.renderer.rules.link_open = function(tokens, idx, options, env, self) {
 
 issues.MetaDataView = Backbone.View.extend({
   el: $("#js-Issue-information"),
-  initialize: function() {
```
Comment (miketaylr) [inline webcompat/static/js/lib/issues.js]: note: this was leftover from the last design. there's nothing to re-render in this view -- it just shows the issue body as-is.

File: webcompat/static/js/lib/issues.js [hunk 2]
```diff
@@ -125,16 +117,12 @@ issues.AsideView = Backbone.View.extend({
 });
 
 issues.BodyView = Backbone.View.extend({
-  el: $(".wc-Issue-report"),
```
Comment (miketaylr) [inline webcompat/static/js/lib/issues.js]: same here: leftovers cleanup.

**Comment by miketaylr** [comment]
Tests busted... will investigate.

**Comment by miketaylr** [comment]
```
FAIL: firefox on any platform - User Activity (auth) - IssueListView renders (11932ms)

NoSuchElement: An element could not be located on the page using the given search parameters.

  at nextTickCallbackWith0Args  <node.js:489:9>

  at process._tickCallback  <node.js:418:13>
```

File: webcompat/static/js/lib/models/milestones.js [hunk 3]
```diff
@@ -4,8 +4,10 @@
 
 var issues = issues || {}; // eslint-disable-line no-use-before-define
 
+//should the milestone model be listening to issue model changes here and reacting here?
```
Comment (miketaylr) [inline webcompat/static/js/lib/models/milestones.js]: This comment should be removed.

File: webcompat/static/js/lib/models/milestones.js [hunk 4]
```diff
@@ -24,22 +26,36 @@ issues.MilestonesModel = Backbone.Model.extend({
     );
     this.set("milestones", orderedMilestones);
   },
-  updateMilestones: function(data) {
-    // prevent talking to server in case we somehow got bogus data
-    if (!_.isObject(data)) {
+
+  // update issueModel.milestone property from here.
```
Comment (miketaylr) [inline webcompat/static/js/lib/models/milestones.js]: This comment should be removed.

**Comment by karlcow** [comment]
ping @MDTsai

**Comment by miketaylr** [comment]
To not block progress on the rest of the issues, I'm just gonna merge this. We can give it a final review before merging in #886.

**Pull request merged**